samples/scaleout/Version_6/Shared/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="NServiceBus" version="5.2.14" targetFramework="net45" />
```

Comment:
Shared between V5 (sender, server) and V6 (worker). I should probably switch to message conventions but that would make the sample more complex.